src/libs/actions/Report.js

```diff
@@ -1448,6 +1449,9 @@ function addPolicyReport(policyID, reportName, visibility, policyMembersAccountI
 
         // The room might contain all policy members so notifying always should be opt-in only.
         CONST.REPORT.NOTIFICATION_PREFERENCE.DAILY,
+        '',
+        '',
+        welcomeMessage,
```

Comment:
Coming from https://github.com/Expensify/App/issues/29612

We didn't ensure that the special characters (like `<`) are HTML-escaped